jib-core/src/main/java/com/google/cloud/tools/jib/global/JibSystemProperties.java

```diff
@@ -23,6 +23,8 @@
 /** Names of system properties defined/used by Jib. */
 public class JibSystemProperties {
 
+  public static final String UPSTREAM_CLIENT = "jib.upstreamClient";
```

Comment:
Sorry for coming late (after vacation). I don't know what "upstream client" means, but because this is an internal property, I prefer `_JIB_UPSTREAM_CLIENT`, like the one `_JIB_DISABLE_USER_AGENT` we have below. Otherwise, LGTM.